oauth-client-credentials/.gitignore

```diff
@@ -0,0 +1 @@
+edge.json
```

Comment:
is this required?

Reply:
Not anymore since we moved from sackmesser to apigeecli, I can remove